block.go

```diff
@@ -34,6 +34,12 @@ func NewBlock(parent Hash, cert QuorumCert, cmd Command, view View, proposer ID)
 	return b
 }
 
+func (b *Block) SetTimestamp(ts time.Time) {
```

Comment:
While touching this file, let's rename `TimeStamp` on Line 85 to the correct spelling: `Timestamp`